instaclustr/resource_encryption_key.go

```diff
@@ -0,0 +1,94 @@
+package instaclustr
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceEncryptionKey() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceEncryptionKeyAdd,
+		Read:   resourceEncryptionKeyRead,
+		Update: resourceEncryptionKeyUpdate,
+		Delete: resourceEncryptionKeyDelete,
+
+		Schema: map[string]*schema.Schema{
+			"key_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"alias": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"arn": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+		},
+	}
+}
+
+func resourceEncryptionKeyAdd(d *schema.ResourceData, meta interface{}) error {
```

Comment:
Could this be renamed to resourceEncryptionKeyCreate for consistency with other resources?